pkg/apis/imageregistry/v1/types.go

```diff
@@ -187,64 +196,89 @@ type ImageRegistryConfigStorageEmptyDir struct {
 // the registry to use the AWS S3 service for backend storage
 // https://docs.docker.com/registry/storage-drivers/s3/
 type ImageRegistryConfigStorageS3 struct {
-	// Bucket is the bucket name in which you want to store the registry's data
-	// Optional, will be generated if not provided
+	// Bucket is the bucket name in which you want to store the registry's
+	// data.
+	// Optional, will be generated if not provided.
+	// +optional
 	Bucket string `json:"bucket"`
-	// Region is the AWS region in which your bucket exists
-	// Optional, will be set based on the installed AWS Region
+	// Region is the AWS region in which your bucket exists.
+	// Optional, will be set based on the installed AWS Region.
+	// +optional
 	Region string `json:"region"`
```

Comment:
@openshift/api-reviewers does it make sense to have an optional field that is not `omitempty`?